tests/codes/test_latex_formula.py

```diff
@@ -0,0 +1,39 @@
+"""Tests for the LatexFormula class."""
+from blueprints.codes.latex_formula import LatexFormula
+
+
+class TestLatexFormula:
+    """Test for LatexFormula."""
+
+    @staticmethod
+    def fixture_1() -> LatexFormula:
```

Comment:
Use @pytest.fixtures instead and put the fixture outside of the testing class.

Reply:
done, great improvement!